bundle-size/app.js

```diff
@@ -649,4 +649,61 @@ module.exports = app => {
 
     response.end();
   });
+
+  // TODO(danielrozenberg): replace the /store with this one, once the amphtml
+  // repo is in sync with this new change.
+  v0.post('/commit/:headSha/store.json', async (request, response) => {
+    const {headSha} = request.params;
+    const {bundleSizes} = request.body;
+
+    if (request.body['token'] !== process.env.TRAVIS_PUSH_BUILD_TOKEN) {
+      return response.status(403).end('You are not Travis!');
+    }
+    if (
+      !bundleSizes ||
+      Object.values(bundleSizes).some(value => typeof value !== 'number')
+    ) {
+      return response
+        .status(400)
+        .end(
+          'POST request to /store must have a key/value object ' +
+            '(string->numeric) field "bundleSizes"'
```

Comment:
Nit: `Map<string, number>` or similar JSDoc format might be clearer

Reply:
Makes sense. Done.